submissions/opeyemiPeter/PHP/nameHobbyLoops.php

```diff
@@ -0,0 +1,10 @@
+<?php
+$mymyNamess = array("Paul", "Sayo", "Faith", "Segun", "Tolulope", "Anuoluwa", "Monisola", "Christine", "Fisayomi");
```

Comment:
Remove this file. The PR for it has been merged @opeyemipeter1759

Reply:
alright, i will do that.